hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java

```diff
@@ -485,7 +483,7 @@ private int batchSize() {
         }
 
         private void checkBatchResults(Object[] results, List<Row> rows)
-                                       throws Throwable {
+                throws Throwable {
```

Comment:
can we keep the origin style: align 'throws' with 'Object[]'

Reply:
#1996